frontend/src/components/pages/EditTeamPage.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useContext, useState } from "react";
+import SampleContext from "../../contexts/SampleContext";
+import MainPageButton from "../common/MainPageButton";
+import SampleContextDispatcherContext from "../../contexts/SampleContextDispatcherContext";
+
+type DeleteButtonProps = { index: number; onClick: (index: number) => void };
+
+const DeleteMemberButton = ({ index, onClick }: DeleteButtonProps) => {
+  return (
+    <button
+      type="button"
+      onClick={() => {
+        onClick(index);
+      }}
+    >
+      Delete
+    </button>
+  );
+};
+
+const EditTeamInfoPage = () => {
+  const { teamName, numTerms, members } = useContext(SampleContext);
+  const dispatchTeamUpdate = useContext(SampleContextDispatcherContext);
+  const [newMember, setNewMember] = useState<string>("");
+
+  const setName = (e: React.FormEvent<HTMLInputElement>) => {
+    dispatchTeamUpdate({ type: "EDIT_NAME", value: e.currentTarget.value });
+  };
+
+  const setNumTerms = (e: React.FormEvent<HTMLInputElement>) => {
+    dispatchTeamUpdate({
+      type: "EDIT_NUM_TERMS",
+      value: parseInt(e.currentTarget.value, 10),
+    });
+  };
+
+  const onMemberDelete = (index: number) => {
+    dispatchTeamUpdate({
+      type: "EDIT_MEMBERS",
+      value: [...members.slice(0, index), ...members.slice(index + 1)],
+    });
+  };
+
+  const onMemberAdd = () => {
+    dispatchTeamUpdate({
+      type: "EDIT_MEMBERS",
+      value: [...members, newMember],
+    });
+  };
+
+  return (
+    <div style={{ maxWidth: "300px", margin: "0 auto", paddingTop: "20px" }}>
+      <h1>Edit Team</h1>
+      <div>
+        Team Name: <input type="text" value={teamName} onChange={setName} />
+      </div>
+      <div style={{ marginTop: "1rem" }}>
+        Num Terms:{" "}
+        <input min={0} type="number" value={numTerms} onChange={setNumTerms} />
+      </div>
+      <div style={{ marginTop: "1rem" }}>
+        Members:
+        {members.map((_name, i) => (
+          <div
+            key={_name}
```

Comment:
Does this work if a name is duplicated? Either way I think it's fine to leave as is since it's just an example